widget/tree.go

```diff
@@ -36,6 +36,8 @@ type Tree struct {
 
 	branchMinSize fyne.Size
 	leafMinSize   fyne.Size
+
+	scroller *ScrollContainer
```

Comment:
Merging develop pushed this in out of order, should it be fixed too?

Reply:
I moved that as part of the conflict fix. Figured it wasn't related to open/selected state, or min sizes, so kept separated